recipes/piler/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: piler
+  version: "0.1"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn:  piler_source.tar.gz
+  url: http://www.drive5.com/piler/piler_source.tar.gz
+  md5: 6f351fb437d0cc6a1ec84f41998621c9
+
+requirements:
+  build:
+  - gcc
```

Comment:
You have both tools in this PR.
I have commended on the other already that it would be wonderful if you could try to also compile it under OSX.

Thanks.